rasa_core/server.py

```diff
@@ -385,7 +385,11 @@ def predict(sender_id):
     @ensure_loaded_agent(agent)
     def log_message(sender_id):
         request_params = request.get_json(force=True)
-        message = request_params.get("text")
+        try:
```

Comment:
@tmbo are you happy with this way of dealing with it? we should probably except the specific error, like `KeyError` or whatever

Reply:
this won't work (there is not going to be a key error, as `.get` never throws one), should be 

```
try:
  message = request_params["message"]
except KeyError:
  message = request_params.get("text")
```

Reply:
Hi @tmbo, Yes! you are right! As I just tried with json.get function. If the value does not exist, python will return "None" instead of throwing an error. Thanks for pointing this out!